broker/src/test/java/org/apache/rocketmq/broker/filter/MessageStoreWithFilterTest.java

```diff
@@ -79,24 +79,17 @@
         try {
             StoreHost = new InetSocketAddress(InetAddress.getLocalHost(), 8123);
         } catch (UnknownHostException e) {
-            e.printStackTrace();
         }
         try {
             BornHost = new InetSocketAddress(InetAddress.getByName("127.0.0.1"), 0);
         } catch (UnknownHostException e) {
```

Comment:
no need to delete  catch part ?

Reply:
Here,  don't  care about this exception, so not hide it .